spec/middleware_integration_spec.rb

```diff
@@ -5,13 +5,16 @@
 
 module RailsEventStore
   RSpec.describe Middleware do
-    DummyEvent = Class.new(RailsEventStore::Event)
```

Comment:
Creating the class in `let` scope will not pollute the global namespace, the class only exists in this spec execution context. Polluting the global namespace might break other specs latter down the chain because a class is already defined for example.